### DISCUSSION TITLE
Leider seit dem letzten Update von Enpal nicht mehr nutzbar

### DISCUSSION THREAD

**Discussion by Busterdick** [gickowtf/enpal-homeassistant#20] [General]
Da Enpal das interface am 18.11. mit einem neuen Update versehen hat, sind nur noch 6 Entities abfragbar. es funktionieren nur noch sesor.enpal_energy_consumption, 
sensor.enpal_energy_external_in_day, 
sensor.enpal_energy_external_out_day, 
sensor.enpal_power_external_total,
sensor.enpal_production_day,
sensor.enpal_solar_production_power

alle anderen Sensoren wie z.B. Battery und Wallbox gehen nicht mehr.

Wird es da wohl eine Anpassung geben?

Grüsse Andreas

**Comment by mardeck** [comment]
Hallo zusammen. Ich würde mich auch über eine Anpassung freuen. Kann man irgendwie helfen? Werden weitere Infos benötigen?

LG
